### PR TITLE
Adding Target Audience Feature

### PR DESCRIPTION
**Pull request by matheusmatos**
Adding new *Target Audience* feature in accord to Issue #13.

screenshot-localhost-3000-2017-03-13-09-02-43

### CODE REVIEW

File: src/views/style.scss [hunk 1]
```diff
@@ -34,4 +38,4 @@
 
 .messagePanel {
   color: gray;
-}
+}
```
Comment (slvnperron) [inline src/views/style.scss]: Missing empty line at the end of the file :)

File: src/views/style.scss [hunk 2]
```diff
@@ -6,6 +6,10 @@
   color: gray;
 }
 
+:global(.form-group) {
```
Comment (slvnperron) [inline src/views/style.scss]: This is going to affect all .form-group (we made this error too on form-control and panel-body). Can you wrap these into a .messenger selector or something to limit these rules only to this module?

File: src/messenger.js [hunk 3]
```diff
@@ -200,6 +200,42 @@ class Messenger extends EventEmitter {
...
+      case 'closeToAll':
+        setting.target_audience.audience_type = 'none'
+        break;
+    }
+
+    return setting;
+  }
+
+  setTargetAudience() {
+    const url = `https://graph.facebook.com/v2.6/me/messenger_profile?access_token=${this.config.accessToken}`
```
Comment (slvnperron) [inline src/messenger.js]: Just wondering why 2.6 instead of 2.7? Also is the new version 2.8?

Comment (matheusmatos) [inline src/messenger.js]: I just followed the example of documentation that's with 2.6. I suggest changing the version API to a param or constant, it's all hard coded and some methods like `_subscribePage()` and `_unsubscribePage()` are using the 2.6 version too.

Comment (matheusmatos) [inline src/messenger.js]: I'll change the version of this request and then we could replace it for a constant.

**Pull request merged**